qxf2_scheduler/routes.py

```diff
@@ -65,73 +68,89 @@ def index():
 
 
 @app.route("/interviewers")
-def listinterviewer():
-    "List the interviewer names,designation"
-    interviewer_work_time_slots = []
-    new_slot = Interviewers.query.join(Interviewertimeslots,Interviewers.interviewer_id==Interviewertimeslots.interviewer_id).values(Interviewers.interviewer_id,Interviewers.interviewer_email,Interviewertimeslots.interviewer_start_time,Interviewertimeslots.interviewer_end_time)
+def listinterviewers():
+    "List all the interviewer names"
+    all_interviewers = Interviewers.query.all()
+    my_interviewers_list = []
+    for each_interviewer in all_interviewers:
+        my_interviewers_list.append({'interviewer_id': each_interviewer.interviewer_id,
+                                     'interviewer_name': each_interviewer.interviewer_name})
+
+    return render_template("list-interviewers.html", result=my_interviewers_list)
+
+
+def parse_interviewer_detail(interviewer_details):
```

Comment:
Rename the method to be `form_interviewer_timeslot(start_time,end_time)`

Reply:
Done